wordpress-mobile/WordPress-iOS
# Fix Quick Start Publish a Post prompt

Fixes #14267. This PR updates the Publish a Post Quickstart tour to reference the new Create FAB.

![qs-1](https://user-images.githubusercontent.com/4780/84376136-802deb00-abd8-11ea-9986-04f66544694e.gif)

Main changes:

* Quick Start tours can now opt to not display Notices for their waypoints. The Publish a Post tour makes use of this to avoid showing a standard notice.
* The waypoint notice description is now posted as part of the element changed notification sent by Quick Start.
* The CreateButtonCoordinator listens for Quick Start notifications, and displays the `.newpost` notice as a tooltip.
* Added a spotlight indicator to the create action sheet if it's displayed during the tour.

**To test:**

* Build and run a fresh install of the app. Log in.
* Once you've logged in, you should see the standard tooltip for the Create FAB
* Activate Quick Start for a site: Me > App Settings > Debug (at the bottom) > Activate Quickstart
* Visit the **Grow Your Audience** Quick Start section and choose **Publish a post**
* When you get back to the blog details screen, you should see the new tooltip on the Create button
* Tap the button and you should see the spotlight indicator on the create sheet
* Tap the sheet to access the editor
* Head back to the Quick Start screen and verify that the tour was completed

PR submission checklist:
